<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow_addmenu</class>
 <widget class="QMainWindow" name="MainWindow_addmenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>517</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>0</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Segoe UI</family>
    <pointsize>16</pointsize>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Add Items</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../img/titleicon.jpeg</normaloff>../img/titleicon.jpeg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_item">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>210</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
background-color: rgb(55, 55, 55);
color: rgb(255, 255, 255);
border style: None;
border-radius: 20px;
}
</string>
    </property>
    <property name="text">
     <string>Item Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_category">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
background-color: rgb(55, 55, 55);
color: rgb(255, 255, 255);
border style: None;
border-radius: 20px;
}
</string>
    </property>
    <property name="text">
     <string>Category</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_category">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>150</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox{
border-style: None;
border-radius: 15px;
background-color: rgb(255, 255, 255);
}

QComboBox QAbstractItemView{
background-color: rgb(255, 255, 255);
}
QComboBox::drop-down{
border-style: None;
border-radius: 15px
}</string>
    </property>
    <item>
     <property name="text">
      <string notr="true">Select Any</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Pizza Mania</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Pizza</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Delicious Sides</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Beverages</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Desserts</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="lineEdit_additem">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>260</y>
      <width>661</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border-style: None;
border-radius: 15px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_additem">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>360</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(255, 159, 51);
border style: None;
border-radius: 20px;
}</string>
    </property>
    <property name="text">
     <string>Add Item</string>
    </property>
    <property name="shortcut">
     <string>Return</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_menu">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>800</width>
      <height>400</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame
{
background-color: rgb(55, 55, 55);
border-radius: 40px;

}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="lineEdit_price">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>90</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
border-style: None;
border-radius: 15px;
}
</string>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_menuitems">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>20</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Add Menu Items</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_category_2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>100</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
background-color: rgb(55, 55, 55);
color: rgb(255, 255, 255);
border style: None;
border-radius: 20px;
}
</string>
    </property>
    <property name="text">
     <string>Price</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_category_3">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>100</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
background-color: rgb(55, 55, 55);
color: rgb(255, 255, 255);
border style: None;
border-radius: 20px;
}
</string>
    </property>
    <property name="text">
     <string>Sub Category</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_category_2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>150</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox{
border-style: None;
border-radius: 15px;
background-color: rgb(255, 255, 255);
}

QComboBox QAbstractItemView{
background-color: rgb(255, 255, 255);
}
QComboBox::drop-down{
border-style: None;
border-radius: 15px
}</string>
    </property>
    <item>
     <property name="text">
      <string notr="true">None</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Simply Veg</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Veg Treat</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Paneer Special</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Veg Feast</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Veg Special</string>
     </property>
    </item>
   </widget>
   <zorder>frame_menu</zorder>
   <zorder>comboBox_category</zorder>
   <zorder>lineEdit_additem</zorder>
   <zorder>pushButton_additem</zorder>
   <zorder>label_category</zorder>
   <zorder>label_item</zorder>
   <zorder>label_menuitems</zorder>
   <zorder>label_category_2</zorder>
   <zorder>label_category_3</zorder>
   <zorder>comboBox_category_2</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
